desktop/caliban.py

```diff
@@ -3825,7 +3825,11 @@ def helper_update_composite(self):
 
     def update_adjusted_raw(self):
         # self.raw may not be in 8 bit (0, 255) format; rescale without modifying original
-        rescaled_raw = rescale_intensity(self.raw[:,:,0:6], in_range = 'image', out_range = 'uint8')
+
+        # get value at 5th and 95th percentile for min and max rescaling, respectively
+        percentiles = np.percentile(self.raw[self.raw > 0], [5, 95])
+        rescaled_raw = rescale_intensity(self.raw[:,:,0:6], in_range=(percentiles[0], percentiles[1]),
+                                         out_range='uint8')
         rescaled_raw = rescaled_raw.astype('uint8')
```

Comment:
out of interest, what was the design choice behind using uint8?

Reply:
The uint8 array gets drawn directly as an RGB image (no colormap gets applied). The underlying raw data type doesn't get changed.